bindings/nodejs/index.js

```diff
@@ -20,6 +20,9 @@
 /// <reference types="node" />
 
 require('dotenv').config()
-const { Operator } = require('./generated.js')
+const { Operator, RetryLayer } = require('./generated.js')
 
 module.exports.Operator = Operator
+module.exports.layers = {
```

Comment:
We don't need expose `Entry`, `Metadata` and `Lister`?

Reply:
They will not be created directly by the user.